Gemfile.lock

```diff
@@ -136,6 +137,10 @@ GEM
     ipaddress (0.8.3)
     jmespath (1.3.1)
     json (2.0.2)
+    kitchen-docker (2.6.0)
```

Comment:
@cdoughty-r7, can you regenerate the Gemfile.lock so `kitchen-docker` is removed from it as well?